src/localforage.js

```diff
@@ -116,7 +116,7 @@ class LocalForage {
         this._dbInfo = null;
 
         this._wrapLibraryMethodsWithReady();
-        this.setDriver(this._config.driver);
+        this.setDriver(this._config.driver).catch(() => {});
```

Comment:
We could probably at least `console.error` this or something if `console.error` exists, just so we don't entirely fail silently (I could see that confusing people in the future).